server/src/test/java/org/elasticsearch/index/mapper/RootObjectMapperTests.java

```diff
@@ -185,4 +185,21 @@ public void testIllegalFormatField() throws Exception {
             assertEquals("Invalid format: [[test_format]]: expected string value", e.getMessage());
         }
     }
+
+    public void testIllegalDynamicTemplates() throws Exception {
+        String mapping = Strings.toString(XContentFactory.jsonBuilder()
+                .startObject()
+                .startObject("type")
+                    .startObject("dynamic_templates")
+                    .endObject()
+                .endObject()
+            .endObject());
+
+        DocumentMapperParser parser = createIndex("test").mapperService().documentMapperParser();
+        MapperParsingException e = expectThrows(MapperParsingException.class,
+                    () -> parser.parse("type", new CompressedXContent(mapping)));
+            assertEquals("Dynamic template syntax error, expects an array of named objects", e.getMessage());
+    }
+}
+
 }
```

Comment:
```suggestion
}
```
This closing bracket is too much now, class closed in line 203 already so the test doesn't compile. Please remove the last two lines so the class compiles.